src/cf-tables/src/cf-tables.less

```diff
@@ -25,13 +28,26 @@
     }
 }
 
-// .table
-
-.table_cell__right-align {
+.o-table_cell__right-align {
     text-align: right;
 }
 
-.table-wrapper__scrolling {
+.o-table__row-links {
+    .respond-to-min( @bp-med-min, {
+        tr:hover {
+            td {
+                background: @table-row-link-bg-hover;
+                color: @table-row-link-color;
+                cursor: pointer;
+              }
+              a {
```

Comment:
Fixed in https://github.com/cfpb/capital-framework/pull/396/commits/d5b92ef4452d9a7a0d5a900b1bc3a28eea9ba8b0